example/hands-on/README.md

```diff
@@ -0,0 +1,290 @@
+# Hands-on
+
+# What is this example doing
+It will show you how to create a project ,service and some test code using Bucky command.
+And will show you how to execute these test code.
+
+## Before starting
+Make sure you have already installed gem "bucky-core"
+```bash
+gen install bucky-core
+```
+
+## 1. Create a new test project
+* Project directory is your working directory when you are executing Bucky test code
+
+Create a new test project.
+```bash
+bucky new hands-on
+```
+You will get a new directory which name is "hands-on" in you current directory.
+
+## 2. Create a new service in project
+* Service means the application name that you are going to test
+* You are able to create multiple services in one project
+* Use snake case for naming
+
+Move into project directory and create a service.
+```bash
+cd hands-on
+bucky make service bucky_hands_on
+```
+You will get a new directory which name is "bucky_hands_on" under services directory.
+
+## 3. Create devices directory, and create pageobject, parts for test code
+* Device only support PC, SP and tablet
+* Test code will be executed in different user agent according to device
+* Pageobject is the file that you can add customize method in
+* Parts is the file that html element of page is declared in
+* Use snake case for naming
+
+Create device directory, pageobject file and part file.
+```bash
+# Create github top page
+bucky make page github_top --service bucky_hands_on --device pc
+# Create github search list page
+bucky make page github_search_list --service bucky_hands_on --device pc
+```
+You will get some new directories and new files like the following:
+```bash
+hands-on/services/bucky_hands_on/
+└── pc
+    ├── pageobject
+    │   ├── github_search_list.yml
+    │   └── github-top.rb
+    └── parts
+        ├── github_search_list.yml
+        └── github-top.yml
+```
```

Comment:
I think to show all tree below hands-on/ dir is better.
Because after making test part and scenario code, modify config file.

Reply:
Thanks,
I'll show every thing in this step.